acct-group/zookeeper/zookeeper-0.ebuild

```diff
@@ -0,0 +1,10 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Group for sys-cluster/zookeeper-bin"
+
+ACCT_GROUP_ID=217
```

Comment:
This uid is already taken. Please will a PR at https://github.com/gentoo/api-gentoo-org/tree/master/files. Use `used_free_uidgids.sh` to get a free uid/gid combo.

Reply:
Lately I think proxy-maint just been picking & updating the uid/gid on merge, PR process delays notably make it harder to keep this in sync too.

Reply:
Yeah it is, I've opened these PR ca 1 month ago

Reply:
Not a problem, I will look later today here and will just reserve some uid/gid.